LogcatCoreLib/src/main/java/info/hannes/logcat/LogfileFragment.kt

```diff
@@ -10,30 +10,21 @@ import info.hannes.timber.FileLoggingTree
 import info.hannes.timber.fileLoggingTree
 import java.io.File
 
-class LogfileFragment : LogBaseFragment(), Observer<String> {
+class LogfileFragment : LogBaseFragment(), Observer<Event<String>> {
```

Comment:
The `Event`  looks like a debounce. The purpose It's not understandable on the name itself. But I've no better suggestion

Reply:
I used the same class name as Google, I decided to leave the class intact so others can relate to it as well in terms of web familiarity, don't know if I'm making sense here, https://github.com/android/architecture-samples/blob/main/app/src/main/java/com/example/android/architecture/blueprints/todoapp/Event.kt